src/styles/index.scss

```diff
@@ -1,3 +1,72 @@
-body {
+@import 'variables';
+@import 'mixin';
+
+@keyframes move {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.page {
   margin: 0;
+  height: 100%;
+
+  @include center_the_element;
+}
+
+.stopwatch {
+  width: $stopwatch_size;
+  height: $stopwatch_size;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  @include center_the_element;
+}
+
+.stopwatch_center {
+  z-index: 1;
+  width: $center_size;
+  height: $center_size;
+  background-color: #f6a603;
+  border-radius: 50%;
+}
+
+.hand {
+  position: absolute;
+
+  &_minutes {
+    position: absolute;
+    transform-origin: bottom;
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    top: -20vmin;
+    left: -1.5vmin;
+
+    animation: move 3600s steps(60) infinite;
+  }
+
+  &_seconds {
+    position: absolute;
+    transform-origin: bottom;
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+
+    top: -38vmin;
+    left: -0.75vmin;
+    animation: move 60s linear infinite;
+  }
+}
+
+.stopwatch--speed-up .hand_seconds {
+  animation-duration: 10s;
+}
+
+.stopwatch--speed-up .hand_minutes {
+  animation-duration: 600s;
 }
```

Comment:
The classes `.hand_seconds` and `.hand_minutes` do not follow the BEM naming convention. They should be named using the format `.block__element--modifier`, e.g., `.hand__seconds--speed-up` and `.hand__minutes--speed-up`. Make sure to update both the class names and the corresponding CSS selectors.